index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Junita's Kitchen</title>
+</head>
+<body>
+  <header>
+<img src="https://3.bp.blogspot.com/-EYljc9Iut4Y/UvGC_45KGMI/AAAAAAAAA9g/kpwmfb1cHPA/s1600/DSC03318.JPG " atl="chhoila photo"
+width="100">
+<nav aria-label="Main site links">
+    <a class="nav-link" href="#About">About</a>
+    <a class="nav-link" href="#Recipe">Soyabean chhoila Recipe</a>
+    <a class="nav-link" href="#contact">Contact</a></nav>
+</header> 
+<main>
+        <h1>About</h1>
+    <section id="about">
+    <p> Soya chunks are made from defatted soy flour, a by-product of extracting soybean oil. It has a protein content which is equal to that of meat. It is quick and also easy to cook.Since I am a vegeterian , it is my one of the favourite foods to eat and cook.I want to dedicate this recipe to my vegetirain friends and for meat lovers too. This is an easy way to make soyabean Choila.</p>
+<img src=" https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJo3YuShJTRNGRhJtZL5XC3hKuHEyaZQc6wg&usqp=CAU" alt="soyabean chhoila phot0" width="300"></section>
+<h2>Soyabean chhoila Receipe:Ingredients</h2>
+<section id="recipe">
+<ul>Soy Chunks (1.5 lbs)</ul>
+ <ul>Onion (Medium sized, finely sliced)</ul> 
+<ul>Serrano Peppers (Green Chilies, 2-3)</ul>
+<ul>Ginger-Garlic Paste (1 table spoon)</ul>
+<ul>Ginger (2 table spoons, thinly sliced)</ul>
+<ul>Garlic (2 table spoons, thinly sliced)</ul>
+<ul>Salt (according to the taste)</ul>
+<ul>Turmeric (1 tea spoon)</ul>
+<ul>Red Chili Powder (1/2 table spoon)</ul>
+<ul>Schezwan Pepper (1 and 1/2 table spoons)</ul>
+<ul>Cumin-Coriander Powder (1 table spoon)</ul>
+<ul>omatoes (3 medium sized, boiled and peeled)</ul>
+<ul>Fenugreek seeds (1 tea spoon)</ul>
+<ul>Oil (2 table spoons)</ul>
+<ul>Lime juice (3 table spoons)/ul></ul></section>
+<h1>Directions:</h1>
+<ol>
+<li>Soak soy chunks on a cold water for 15 minutes.</li>
+<img src=" https://1.bp.blogspot.com/-vhIjSXOXcsk/UvF9ST8dXfI/AAAAAAAAA8w/k0YDKotbTuo/s1600/DSC03309.JPG" width="200">
+<li>Heat pan and add a tea spoon of oil and fry soya chunks until red and crispy.</li>
+<img src=" https://4.bp.blogspot.com/-5ghySqG2YAY/UvF9WHgCXoI/AAAAAAAAA84/QYpJ26rdhDQ/s1600/DSC03311.JPG" width="200">
+<li>On a large bowl, place soya chunks, add all of the above mentioned ingredients except fenugreek seeds, lime juice and remaining oil.</li>
+<img src="https://2.bp.blogspot.com/-sMPBM3oBqyU/UvF9ultY6xI/AAAAAAAAA9A/CZRa-4EqGHw/s1600/DSC03314.JPG" width="200">
```

Comment:
better to down load the image into your project than to link to some other websites like `https://2.bp.blogspot.com/-sMPBM3oBqyU/UvF9ultY6xI/AAAAAAAAA9A/CZRa-4EqGHw/s1600/DSC03314.JPG`